WootricSDK/WootricSDK/WTRiPADFeedbackView.m

```diff
@@ -253,4 +266,16 @@ - (NSDictionary *)selectedAnswers {
   return answers;
 }
 
+- (NSArray *)shuffleArray:(NSArray *)array {
```

Comment:
Duplicate function. Same function is present in - WTRFeedbackView.m file, You can reuse same function by keeping in some utility class.